app/views/schools/_form.html.erb

```diff
@@ -6,6 +6,17 @@
       required: false, id: 'school_name' %>
   </div>
 
+  <div class="form-group row">
+  <div class='col-12'>
+    <%= f.label :country, "Country", class: "label-required" %>
+    <%= f.country_select(
+      :country,
+      { priority_countries: ['United States'], include_blank: "Select an option" },
```

Comment:
Can we add:

```
priority_countries_divider: "------"
```

Reply:
The United States is already separated by the "--------" divider